src/components/views/SuggestedViews/ViewCard.tsx

```diff
@@ -35,7 +36,7 @@ const ViewCard: React.FunctionComponent<ViewCardProps> = ({ onClick,  view }) =>
                         </Grid>
                         <Grid item>
                             <Typography component="p" variant="caption">
-                                Created by { view.owner.name }
+                                <FormattedMessage id="misc.views.suggested.created" />{ ' ' }{ view.owner.name }
```

Comment:
Please use `<FormattedMessage id="…" values={{ name: view.owner.name }}/>` to interpolate the name into the localized string, as not all languages will have it at the end.

Reply:
Was just doing that before I read this :-D